src/EnergyPlus/PhotovoltaicThermalCollectors.cc

```diff
@@ -233,7 +272,81 @@ namespace PhotovoltaicThermalCollectors {
                                                                      state.dataIPShortCut->cNumericFieldNames);
             if (UtilityRoutines::IsNameEmpty(state, state.dataIPShortCut->cAlphaArgs(1), state.dataIPShortCut->cCurrentModuleObject, ErrorsFound))
```

Comment:
Same set of comments as previous function.

Reply:
romoved